src/libraries/Error.sol

```diff
@@ -185,6 +185,9 @@ error InitParamZero(string name);
 /// @dev Thrown when data hashes where expected but not where present on the tx
 error MissingDataHashes();
 
+/// @dev Batch Data validation over avail bridge failed with the given merkle proof input
+error BatchDataValidationForAvailDAFailed(bytes32);
+
```

Comment:
should be a part of the contract / module for the avail DA and fetch these constants from there